schedule/yast/autoyast/autoyast_tftp.yaml

```diff
@@ -6,9 +6,12 @@ description: >
      - Post installation script execution
      - Adding host entries to the /etc/hosts
      - ntp/chrony configuration
+vars:
+  AUTOYAST: autoyast_sle15/autoyast_tftp.xml
+  AUTOYAST_VERIFY: autoyast_sle15/tftp.sh
```

Comment:
I see that https://github.com/yast/aytests-tests/blob/master/aytests/tftp.list contains more checks. Did you exclude them on purpose as irrelevant? Or shall we keep those?

Reply:
No, I got confused with the setting in testsuite, where we had AUTOYAST_VERIFY=aytests/tftp.sh instead of tftp.list.

Reply:
you can check previous work related with settings [here](https://openqa.suse.de/tests/4840257/settings/schedule/yast/autoyast/autoyast_lvm.yaml)

Reply:
Ah, I see. Then some parts might not work, so let's test if they do, if not, we are not reducing the coverage.